lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts

```diff
@@ -72,7 +72,7 @@ export class AuthBearerInterceptor implements HttpInterceptor {
     // prevent adding any content type, to properly handle formData with boundary browser generated value,
     // as adding any Content-Type its going to break the upload functionality
 
-    if (headers.get('Content-Type') === 'multipart/form-data') {
+    if (headers.get('Content-Type') === 'multipart/form-data' && !(reqBody instanceof FormData)) {
```

Comment:
Did you try if it still work on apa?

Reply:
nop, I am going to test it on apa as well 👍

Reply:
thanks 🙏  so we're sure that works on both

Reply:
Tested all apps with adf configuration, looks good on local:

- workspace hxp - OK
- content-ee-apa - OK
- studio-hxp - OK
- modeline-ee - OK
- admin-hxp - OK
- admin-apa - OK